modules/aws/sqs/variables.tf

```diff
@@ -0,0 +1,108 @@
+variable "create" {
+  description = "Whether to create SQS queue"
+  type        = bool
+  default     = true
+}
+
+variable "name" {
```

Comment:
@suryaoruganti what inputs do we want from users for SQS? I am thinking of getting

name
fifo_queue
max_message_size
message_retention_seconds
visibility_timeout_seconds

redrive_allow_policy

Reply:
these params are not in the `terragrunt.hcl` file

Reply:
These 3 Parameters already there in variables.tf.
max_message_size
message_retention_seconds
visibility_timeout_seconds

Reply:
redrive_allow_policy 
Added and tested